WordPressVIPMinimum/Sniffs/Plugins/ZoninatorSniff.php

```diff
@@ -78,7 +78,7 @@ public function process( File $phpcsFile, $stackPtr ) {
 		$version = $this->remove_wrapping_quotation_marks( $tokens[ $version ]['content'] );
 
 		if ( true === version_compare( $version, '0.8', '>=' ) ) {
-			$phpcsFile->addWarning( 'Zoninator of version >= v0.8 requires WordPress core REST API. Please, make sure the `wpcom_vip_load_wp_rest_api()` is being called on all sites loading this file.', $stackPtr, 'Zoninator' );
+			$phpcsFile->addWarning( 'Zoninator of version >= v0.8 requires WordPress core REST API. Please, make sure the `wpcom_vip_load_wp_rest_api()` is being called on all sites loading this file.', $stackPtr, 'Found' );
```

Comment:
Should we have something more descriptive than `Found` for the Zoninator version warning?

Reply:
Right now, all the sniff does is check that Zoninator is present, and not the default version. I'm open to suggestions for a different violation code.

Reply:
Maybe something like `GreaterThanVersion`?

Reply:
There might be a version of Zoninator that doesn't require the REST API, so instead of checking for >= 0.8, the check might be >= 0.8 && <= 1.2, in which case a `GreaterThanVersion` name wouldn't make sense.

I've gone for `RequiresRESTAPI`, since that it was this message is about - the fact it's checking the version is just the implementation, not the intent.